### PR TITLE
Fixes for pydocstyle rule D211

### PR DESCRIPTION
**Pull request by morrme**
Ref. #1170

### CODE REVIEW

**Comment by morrme** [comment]
@peterjc I resolved the style issues but the check is still failing...

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": false
+}
```
Comment (peterjc) [inline .vscode/settings.json]: Is vscode your editor? Can you remote this setting file please.

We would expand ``.gitignore`` if that helps?

File: Bio/SearchIO/HmmerIO/hmmer3_domtab.py [hunk 2]
```diff
@@ -6,10 +6,8 @@
 """Bio.SearchIO parser for HMMER domain table output format."""
 
 from itertools import chain
-
```
Comment (peterjc) [inline Bio/SearchIO/HmmerIO/hmmer3_domtab.py]: These blank lines are unrelated to D211, and indeed this one is deliberate to split standard library imports from Biopython ones. Flake8 has optional checks to enforce this kind of import rule but we don't currently follow that.

File: Bio/SearchIO/HmmerIO/hmmer3_domtab.py [hunk 3]
```diff
@@ -70,6 +68,7 @@ def _parse_row(self):
 
         return {'qresult': qresult, 'hit': hit, 'hsp': hsp, 'frag': frag}
 
+
```
Comment (peterjc) [inline Bio/SearchIO/HmmerIO/hmmer3_domtab.py]: There should be only one blank line between methods

**Comment by peterjc** [comment]
It looks like you've removed some D211 lines but then put them back again - I think the merge went wrong.

It might be simplest to start a new branch and cherry-pick ee683d167edfc7f3484149e7baa0e8b687a394ca and then fix the accidental changes that included (some variable name typos etc)?

File: .vscode/settings.json (hunk 1 above)
Comment (morrme) [inline .vscode/settings.json]: @peterjc I just caught that in the most recent set of fixes. It seems that if I hit "ignore" on one of VSCode's popups, it adds a setting to remember my choice. Sorry about that! I will remove it.  It is pretty simple to check for it before committing and get rid of it.

File: Bio/SearchIO/HmmerIO/hmmer3_domtab.py [hunk 4]
```diff
@@ -169,14 +167,14 @@ class Hmmer3DomtabHmmhitIndexer(Hmmer3TabIndexer):
     coordinates are hit coordinates."""
 
     _parser = Hmmer3DomtabHmmhitParser
-    _query_id_idx = 3
+    _qury_id_idx = 3
```
Comment (peterjc) [inline Bio/SearchIO/HmmerIO/hmmer3_domtab.py]: This looks like an accidentally committed typo (removing the e)

File: Bio/SearchIO/HmmerIO/hmmer3_domtab.py [hunk 5]
```diff
@@ -169,14 +167,14 @@ class Hmmer3DomtabHmmhitIndexer(Hmmer3TabIndexer):
...
 
 
 class Hmmer3DomtabHmmqueryIndexer(Hmmer3TabIndexer):
     """Indexer class for HMMER domain table output that assumes HMM profile
     coordinates are query coordinates."""
 
-    _parser = Hmmer3DomtabHmmqueryParser
+    _paser = Hmmer3DomtabHmmqueryParser
```
Comment (peterjc) [inline Bio/SearchIO/HmmerIO/hmmer3_domtab.py]: This looks like an accidentally committed typo (removing the r)

File: Bio/SearchIO/HmmerIO/hmmer3_domtab.py [hunk 6]
```diff
@@ -296,7 +294,7 @@ def _build_row(self, qresult):
                 qaccw, qresult_acc, qresult.seq_len, hit.evalue, hit.bitscore,
                 hit.bias, hsp.domain_index, len(hit.hsps), hsp.evalue_cond, hsp.evalue,
                 hsp.bitscore, hsp.bias, hmm_from, hmm_to, ali_from, ali_to,
-                hsp.env_start + 1, hsp.env_end, hsp.acc_avg, hit.description)
+               hsp.env_start + 1, hsp.env_end, hsp.acc_avg, hit.description)
```
Comment (peterjc) [inline Bio/SearchIO/HmmerIO/hmmer3_domtab.py]: This looks like an accidentally committed typo (removing a space and breaking the indentation alignment)

**Comment by morrme** [comment]
@peterjc I am still pretty new to this, and I think I set up the branch for 301 incorrectly, which may have resulted in some issues. I think I got the hang of it and did a little better with #1176 .

File: Bio/SearchIO/HmmerIO/hmmer3_domtab.py (hunk 4 above)
Comment (morrme) [inline Bio/SearchIO/HmmerIO/hmmer3_domtab.py]: These are so strange, as I didn't have to touch the code beyond the blank lines.

**Comment by peterjc** [comment]
On GitHub, near the top of the pull request page are three tabs: Conversation, Commits, Files Changed.

The "Files Changed" view gives a nice overview of the net changes your pull request is making. Right now all the deliberate changes for D211 are missing - although I'm not quite sure when it was most likely a merge.

In general there is no need for you to do a merge while working on a pull request.

File: Bio/SearchIO/HmmerIO/hmmer3_domtab.py (hunk 4 above)
Comment (peterjc) [inline Bio/SearchIO/HmmerIO/hmmer3_domtab.py]: I sometimes find I have made accidental changes from mis-typed keyboard shortcuts. It happens.

**Comment by morrme** [comment]
@peterjc I believe I made 301 a "child" of 211, then I later had an error when pushing that required me to pull first. I noticed it when I did the PR for 301 and there were commits from the 211 PR in there. I am so sorry for the mess! I will clean it up.

**Comment by peterjc** [comment]
This was applied as https://github.com/biopython/biopython/commit/4f5a00e6d9d8c924ddf373624d1a9b38f1e31ed8
